packages/plugin-ext/src/hosted/node/hosted-plugin-process.ts

```diff
@@ -55,7 +55,7 @@ export class HostedPluginProcess implements ServerPluginRunner {
     @named(PluginHostEnvironmentVariable)
     protected readonly pluginHostEnvironmentVariables: ContributionProvider<PluginHostEnvironmentVariable>;
 
-    @inject(MessageService)
+    @inject(MessageService) @optional()
```

Comment:
I wonder whether we should not simply bind a `MessageService` in the headless plugin case so that messages are properly logged and not completely lost. Simply writing out to the console like the default `MessageClient` and `MessageService` should be enough, I think.

Reply:
The goofy part is that we already have a `MessageService` bound. So this doesn't need to be optional.

Reply:
Yay! :D